docs/src/explanations/memories.md

```diff
@@ -174,3 +176,146 @@ class MaskedRWSmem extends Module {
 }
 ```
 
+### Memory Initialization
```

Comment:
does this belong here, or in the `Experimental Features` chapter?

Reply:
Ah maybe... since it uses features from `chisel3.utils.experimental` right?

Reply:
Even moving there, don't you think it's good to have a reference on how to initialize memory from this page to there? Since people looking into it will not look for it in experimental.

Reply:
well there is already https://www.chisel-lang.org/chisel3/docs/appendix/experimental-features.html#loading-memories...

Reply:
ah, i see your point. Yes, i think a cross reference to the experimental features page makes a lot of sense.

Reply:
Gonna move most information to the experimental page and put a reference to it.
Also I'll fix some formatting in it.